db/migrate/20220120004635_create_post_tags.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Migration create post_tags table in database.
+class CreatePostTags < ActiveRecord::Migration[6.1]
+  def change
+    create_table :post_tags do |t|
+      t.belongs_to :post
+      t.belongs_to :tag
+
+      t.timestamps
+    end
```

Comment:
In order to have table set up after `create_table` block you can add this line `add_index :post_tags, %i[post_id tag_id], unique: true`